api_calls/reference/rental_contacts/public-scope-response.json

```diff
@@ -0,0 +1,28 @@
+{ "links": {
+    "rental_contacts.rental": "http://test.host/api/v3/rentals/{rental_contacts.rental}",
```

Comment:
the URLs should point to https://bookingsync.com

Reply:
good catch!